homeassistant/components/history.py

```diff
@@ -185,8 +216,77 @@ def get(self, request, datetime=None):
         end_time = start_time + one_day
         entity_id = request.args.get('filter_entity_id')
 
-        return self.json(
-            get_significant_states(start_time, end_time, entity_id).values())
+        return self.json(get_significant_states(
+            start_time, end_time, entity_id, Filters(self.config)).values())
+
+
+# pylint: disable=too-few-public-methods
+class Filters(object):
+    """Container for the configured include and exclude filters."""
+
+    def __init__(self, config):
```

Comment:
I would prefer the `Filters` class to not be aware of the shape of the config. That way we could potentially re-use this class.